libs/components/indicators/src/lib/modules/icon/icon-resolver.service.ts

```diff
@@ -22,6 +22,10 @@ export class SkyIconResolverService {
   constructor() {
     // Map the icons by name for more efficient lookup.
     for (const glyph of this.#manifestSvc.getManifest().glyphs) {
+      // TODO: remove after pulling in new Icons
+      if (glyph.faName && !glyph.faNames) {
+        glyph.faNames = [glyph.faName];
+      }
```

Comment:
is this TODO still relevant? If so we might need some more context as to why this is the workaround for not having the latest icons. Is it populating the value in faNames with what is in faName? these variables are hard to understand the difference

Reply:
This is essentially just defensive coding. The icons library was updated so all instances of `faName` were converted to `faNames` arrays, but since it's only a deprecated property and not actually removed this should stay until it no longer exists.

Reply:
aaah gotcha. ok i think we should clarify that in the TODO so it's clear exactly when we will be removing it. potentially with a link to a story that marks when we do remove it. other than that, lgtm

Reply:
updated